builder/image/templates/builder

```diff
@@ -101,7 +101,7 @@ if [ ! -f Dockerfile ]; then
     if [ -f /buildpacks ]; then
         BUILD_OPTS+=' -v /buildpacks:/tmp/buildpacks:rw'
         # give non-root slugbuilder user R/W perms for docker volumes
-        chown -R 2000:2000 /buildpacks
+        sudo chown -R 2000:2000 /buildpacks
```

Comment:
I changed it to allow this single command, since it's not possible for the git user to do this without sudo. See: [https://superuser.com/questions/697608/chown-operation-not-permitted](https://superuser.com/questions/697608/chown-operation-not-permitted)

`+RUN echo "%git    ALL=(ALL:ALL) NOPASSWD:/bin/chmod -R 2000\:2000 /buildpacks" >> /etc/sudoers`